### PR TITLE
Document how to turn on the extension feature

### PR DESCRIPTION
**Pull request by alexevanczuk**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,6 +15,8 @@ group :development do
 end
 ```
 
+Make sure the relevant features are enabled in `rubyLsp.enabledFeatures` in your VSCode settings, such as `codeLens`. See https://shopify.github.io/ruby-lsp for the full list of configuration options.
```
Comment (st0012) [inline README.md:18]:
```suggestion
> [!IMPORTANT] 
> Make sure the relevant features are [enabled](https://github.com/Shopify/ruby-lsp/tree/main/vscode#enable-or-disable-features) under your VSCode's `rubyLsp.enabledFeatures` setting, such as `codeLens`.
```

Comment (alexevanczuk) [reply README.md]: Thanks! Committed

**Comment by alexevanczuk** [comment]
@st0012 Applied your suggestion! Good to merge?

**Review by st0012 — APPROVED** [review]
Thanks 😄

**Pull request merged**